javalib/src/main/scala/java/lang/MathJDK8Bridge.scala

```diff
@@ -21,54 +21,52 @@
 package java.lang
 
 private[java] object MathJDK8Bridge {
-  def addExact(a: scala.Int, b: scala.Int): scala.Int = sys.error("stub")
+  def addExact(a: scala.Int, b: scala.Int): scala.Int = ???
```

Comment:
So the idea is that you replace `sys.error("stub")` with `???` if it is a bridge, and with `new jl.Error("stub")` if it is a primitive? I think using `???` is not advisable, since it throws a `NotImplementedError` which is not really appropriate here (the problem is a messed up classpath, not the implementation).

I suggest:
- Replacing these with `jl.Error("stub")`.
- or replacing `sys.error("stub")` with `jl.Error("primitive")` / `jl.Error("bridge")` as appropriate (might be overkill?).